### PR TITLE
Issue1434 invariant documentation

### PR DESCRIPTION
**Pull request by smk78**
I have rather significantly revised invariant_help.rst

Do you approve?

### CODE REVIEW

File: src/sas/sasgui/perspectives/invariant/media/invariant_help.rst [hunk 1]
```diff
@@ -1,135 +1,210 @@
...
-*  The Guinier function $I_0 exp(-R_g^2 Q^2/3)$ where $I_0$
-   and $R_g$ are obtained by fitting as for the high-\ $Q$ region above.
-   Alternatively a power law can be used.
+.. math::
 
-.. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
+    Q^* = {2 \pi^2 (\Delta\rho)^2 \phi_1 \phi_2}
+    
+where $\Delta\rho = (\rho_1 - \rho_2)$ is the SLD contrast and $\phi_1$ and
+$\phi_2$ are the volume fractions of the two phases ($\phi_1$ + $\phi_2$ = 1).
```
Comment (pkienzle) [inline src/sas/sasgui/perspectives/invariant/media/invariant_help.rst]: Format as one equation: `$\phi_1 + \phi_2 = 1$` ?

File: src/sas/sasgui/perspectives/invariant/media/invariant_help.rst [hunk 2]
```diff
@@ -1,135 +1,210 @@
...
+SLD Fluctuation
 ^^^^^^^^^^^^^^^
 
-The volume fraction $\phi$ is related to $Q^*$  by
+The mean-square average of the SLD fluctuation is
 
 .. math::
 
-    \phi(1 - \phi) = \frac{Q^*}{2\pi^2(\Delta\rho)^2} \equiv A
+    <\eta^2> = < ((\rho^*)^2 > - < (\rho^*) >^2
```
Comment (pkienzle) [inline src/sas/sasgui/perspectives/invariant/media/invariant_help.rst]: Use ```\langle \eta^2 \rangle = \langle  ((\rho^*)^2 \rangle - \langle (\rho^*) \rangle^2```

File: src/sas/sasgui/perspectives/invariant/media/invariant_help.rst [hunk 3]
```diff
@@ -1,135 +1,210 @@
...
 
-where $\Delta\rho$ is the SLD contrast.
+where
 
 .. math::
 
-    \phi = \frac{1 \pm \sqrt{1 - 4A}}{2}
+    < (\rho^*)^2 > = \phi_1 \rho_1^2 + \phi_2 \rho_2^2
```
Comment (pkienzle) [inline src/sas/sasgui/perspectives/invariant/media/invariant_help.rst]: `\langle (\rho^*)^2 \rangle = \phi_1 \rho_1^2 + \phi_2 \rho_2^2`

File: src/sas/sasgui/perspectives/invariant/media/invariant_help.rst [hunk 4]
```diff
@@ -1,135 +1,210 @@
...
+
+.. math::
+    
+    < (\rho^*) > = \phi_1 \rho_1 + \phi_2 \rho_2
```
Comment (pkienzle) [inline src/sas/sasgui/perspectives/invariant/media/invariant_help.rst]: `\langle (\rho^*) \rangle = \phi_1 \rho_1 + \phi_2 \rho_2

File: src/sas/sasgui/perspectives/invariant/media/invariant_help.rst (hunk 1 above)
Comment (smk78) [reply src/sas/sasgui/perspectives/invariant/media/invariant_help.rst]: Done

File: src/sas/sasgui/perspectives/invariant/media/invariant_help.rst (hunk 2 above)
Comment (smk78) [reply src/sas/sasgui/perspectives/invariant/media/invariant_help.rst]: Done

File: src/sas/sasgui/perspectives/invariant/media/invariant_help.rst (hunk 3 above)
Comment (smk78) [reply src/sas/sasgui/perspectives/invariant/media/invariant_help.rst]: Done

File: src/sas/sasgui/perspectives/invariant/media/invariant_help.rst (hunk 4 above)
Comment (smk78) [reply src/sas/sasgui/perspectives/invariant/media/invariant_help.rst]: Done

**Comment by smk78** [comment]
Made latex improvements suggested by @pkienzle

**Comment by smk78** [comment]
NB: when merged this file will go into 4.2.2 Master and will therefore also need copying to the ESS_GUI branch.

**Comment by smk78** [comment]
Given I now have both the 5.x and 4.x docs online and we have been hauled up on our Invariant help page, I'm going to make an executive decision here and merge my own changes on the grounds that whilst @butlerpd might not entirely happy with my wording, they are an improvement, and can always be tweaked further if needed.

**Pull request merged**